### PR TITLE
remove Jabber & Mattermost

### PR DESCRIPTION
**Pull request by kritzl**

### CODE REVIEW

**Review by ftsell — CHANGES_REQUESTED** [review]
please merge into staging before going to master

File: locale/en_US/LC_MESSAGES/django.po [hunk 1]
```diff
@@ -2093,10 +1991,10 @@ msgstr "Change Email"
 #: mafiasi/registration/templates/registration/account.html:92
 msgid ""
 "Note: You can \"change\" your password to the current password to "
-"synchronize it with other services (e.g. Jabber)"
```
Comment (timonegk) [inline locale/en_US/LC_MESSAGES/django.po:2096]: This also requires a change in the registration/account.html template. (Actually, the note can be removed there, we don't need any kind of synchronization anymore.)

Comment (timonegk) [inline locale/en_US/LC_MESSAGES/django.po:2096]: This also requires a change in the registration/account.html template. (Actually, the note can be removed there, we don't need any kind of synchronization anymore.)

**Comment by timonegk** [comment]
Also please rebase the PR on the current staging branch.

File: locale/en_US/LC_MESSAGES/django.po (hunk 1 above)
Comment (kritzl) [reply locale/en_US/LC_MESSAGES/django.po:2096]: The Note was already removed by 82c1c67

**Comment by ftsell** [comment]
if your rebase onto the newest staging branch, test execution has been fixed and should work now

**Review by ftsell — APPROVED** [review]

**Pull request merged**